### PR TITLE
autotest:Copter test fix for some frames

### PR DESCRIPTION
**Pull request by arshPratap**
provided a solution for some of the frames listed here #17318.
Thanks !!

### CODE REVIEW

File: Tools/autotest/default_params/copter-octaquad-cwx.parm [hunk 1]
```diff
@@ -0,0 +1 @@
+FRAME_TYPE     14
```
Comment (khancyr) [inline Tools/autotest/default_params/copter-octaquad-cwx.parm:1]: I doubt that is the same as hexa

**Review by peterbarker — CHANGES_REQUESTED** [review]
@arshPratap please correct the frame types.  You're after the `motor_frame_type` enumeration from `AP_Motors_Class.h`

**Comment by peterbarker** [comment]
Nudge @arshPratap

**Comment by arshPratap** [comment]
> @arshPratap please correct the frame types. You're after the `motor_frame_type` enumeration from `AP_Motors_Class.h`

@peterbarker  Sir, am a little confused with the exact configuration to use.Based on the #17318 , I thought we had to use the frame type configuration for each file.

**Comment by jmachuca77** [comment]
Looks like you need both FRAME_CLASS and FRAME_TYPE parameters 
https://ardupilot.org/copter/docs/parameters.html#frame-class-frame-class
https://ardupilot.org/copter/docs/parameters.html#frame-type-frame-type-x-v-etc

**Comment by arshPratap** [comment]
I have made the suggested changes. @peterbarker  sir, please take a look thanks

**Review by peterbarker — APPROVED** [review]
LGTM

**Comment by peterbarker** [comment]
Merged, thanks!

**Pull request merged**